### ISSUE TITLE
Messed up dataset

### ISSUE THREAD

**Issue by rhjdzuDL** [SketchyScene/SketchyScene#21]
As stated, there are 5617 samples for training set, 535 for validation set, and 1113 for test set.

However, some `reference` images are missing:
- only `5616` references are presented in training set 
- and only `530` references in validation set 

There's also something wrong with the `records`:
- there are `5618` json records in training set
- `541` json records in validation set
 

I hope you guys can check it again, and be careful of misalignment. :smiley:

**Comment by MarkMoHR** [comment]
Hi @rhjdzuDL ,

Thanks for your interest in this repo.

Regarding the `records`, there are some duplicated files (which are not misaligned because the number of the jsons are not aligned to any sketches). I have removed them and updated the `records` package.

Regarding the `reference`, some reference images are missing because those corresponding sketches are not generated based on the references, but by users freely (with paired semantic and instance label maps). So if you want to use the references, I recommand using the reference images provided in the current package and their corresponding sketches.
